src/DateRangePickerInput/web/DateRangePickerInput.js

```diff
@@ -29,6 +30,10 @@ class DateRangePickerInput extends React.Component {
     };
     this.datePickerHasFocus = false;
     this.timeout = {};
+    this.range = {
+      min: this.props.range.min >= 0 ? this.props.range.min : 1,
```

Comment:
I would rather have this in `render` than here since we should not block  props reflow

Reply:
How about this?
```
makeSelectionRange = (selectionRange) => ({
    min: selectionRange.hasOwnProperty('min') ? selectionRange.min : 1,
    max: selectionRange.hasOwnProperty('max') ? selectionRange.max : Infinity,
  })
```
and from render you can call this 
`const selectionRange = makeSelectionRange(this.props.selectionRange)`